src/AppServices/KafkaPage/KafkaPageV2.tsx

```diff
@@ -64,13 +64,14 @@ export const KafkaPageV2: FunctionComponent = () => {
                 component="a"
                 href="https://marketplace.redhat.com/en-us/products/red-hat-openshift-streams-for-apache-kafka"
                 target="_blank"
+                ouiaId="link-purchase-now"
```

Comment:
The Jira task suggested adding data-ouia-id to the link, but I am really not sure the link refers to the button component